### ISSUE TITLE
Numpy failing to install with python3.5

### ISSUE THREAD

**Issue by dstuck** [numpy/numpy#16294]
The new release of numpy fails to install when using python3.5. It seems clear the problem is the use of f string formatting in the setup.py file that is only supported in python 3.6 and higher.

### Reproducing code example:

```
docker run python:3.5.2-alpine pip install numpy==1.19.0rc1
```

### Error message:

      File "/tmp/pip-build-gg798dty/numpy/setup.py", line 182
        raise ValueError(f'Submodule {p} missing')
                                                ^
    SyntaxError: invalid syntax

### Numpy/Python version information:

Fails installing `numpy==1.19.0rc1`

**Comment by charris** [comment]
NumPy 1.19.x doesn't support Python 3.5, only 3.6-3.8

**Comment by dstuck** [comment]
Ok, then this metadata should probably be updated: https://github.com/numpy/numpy/blob/0ea1a6e4f7904b79d26239804c1712a389f46a6b/setup.py#L448

**Comment by charris** [comment]
Fixed in #16295. Thanks for the heads up.
